contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -135,107 +337,345 @@ NXmpes(NXobject):
+            doc: |
+              In the case of an experiment in which the gas pressure changes and is recorded,
+              this is an array of length m of gas pressures.
+      flood_gun(NXactuator):
+        exists: optional
+        doc: |
+          Device to bring low-energy electrons to the sample for charge neutralization
+        name:
+          exists: recommended
+        physical_quantity:
+          enumeration: [current]
+        type:
+          exists: optional
+        current(NX_FLOAT):
           exists: recommended
           unit: NX_CURRENT
+          doc: |
+            In case of a fixed or averaged electron current, this is the scalar current.
+            It can also be an 1D array of output current (without time stamps).
+        current_log(NXlog):
+          exists: optional
+          value(NX_NUMBER):
+            unit: NX_CURRENT
+            doc: |
+              In the case of an experiment in which the electron current is changed and
+              recorded with time stamps, this is an array of length m of current setpoints.
     (NXprocess):
+      exists: recommended
       doc: |
         Document an event of data processing, reconstruction, or analysis for this data.
         Describe the appropriate axis calibrations for your experiment using one or more
         of the following NXcalibrations
       energy_calibration(NXcalibration):
         exists: optional
-        applied(NX_BOOLEAN):
-          doc: |
-            Has an energy calibration been applied?
+        doc: |
+          Calibration event on the energy axis.
+          
+          For XPS, the calibration should ideally be performed according to
+          `ISO 15472:2010`_ specification.
+          
+          .. _ISO 15472:2010: https://www.iso.org/standard/74811.html
         calibrated_axis(NX_FLOAT):
           exists: recommended
           doc: |
             This is the calibrated energy axis to be used for data plotting.
       angular_calibration(NXcalibration):
         exists: optional
-        applied(NX_BOOLEAN):
-          doc: |
-            Has an angular calibration been applied?
         calibrated_axis(NX_FLOAT):
           exists: recommended
           doc: |
             This is the calibrated angular axis to be used for data plotting.
       spatial_calibration(NXcalibration):
         exists: optional
-        applied(NX_BOOLEAN):
-          doc: |
-            Has an spatial calibration been applied?
         calibrated_axis(NX_FLOAT):
           exists: recommended
           doc: |
             This is the calibrated spatial axis to be used for data plotting.
       momentum_calibration(NXcalibration):
         exists: optional
-        applied(NX_BOOLEAN):
-          doc: |
-            Has an momentum calibration been applied?
         calibrated_axis(NX_FLOAT):
           exists: recommended
           doc: |
             This is the momentum axis to be used for data plotting.
```

Comment:
It appears a bit inconsitent that we state all these calibrations, while only requiring an energy axis. This should eventually be handled like NXdata_mpes (i.e. NXcalibration_mpes)

Reply:
I guess they should go to NXprocess_mpes, right?

Reply:
We could make an `NXprocess_mpes` extending `NXprocess` and then state all the details of the calibration there.
Then in NXmpes, we use `NXprocess_mpes` with only `energy_calibration` as optional.

Reply:
That was the idea, exactly, but requires base class inheritance. We can anyways still prepare it, and mention names in the docstring, like for NXdata.

Reply:
I have added now a `NXprocess_mpes` base class, in line with the base class inheritance in `NXdata_mpes_*`.

For me, it does not make sense to remove the special NXcalibration events from `NXmpes` for now and _just_ mention them in the docstrings. If we do it like that, we lose all control of how we want to record calibration events _right now_. I would only remove them once we start to use `NXprocess_mpes` with base class inheritance.

Reply:
fine with me.